Source/WebCore/animation/AnimationTimelinesController.h

```diff
@@ -100,6 +100,7 @@ class AnimationTimelinesController final : public CanMakeCheckedPtr<AnimationTim
     Vector<Ref<ScrollTimeline>>& timelinesForName(const AtomString&);
     Vector<WeakPtr<Element, WeakPtrImplWithEventTargetData>> relatedTimelineScopeElements(const AtomString&);
     void attachPendingOperations();
+    bool isPendingTimelineAttachment(WebAnimation&);
```

Comment:
I expect the argument can be `const`, and so can the method itself.

Reply:
Fixed.